Networking/Networking/Remote/AccountRemote.swift

```diff
@@ -114,6 +130,46 @@ public class AccountRemote: Remote, AccountRemoteProtocol {
 
         enqueue(request, mapper: mapper, completion: completion)
     }
+
+    public func loadUsernameSuggestions(from text: String) async -> [String] {
+        let path = Path.usernameSuggestions
+        let parameters = [ParameterKey.name: text]
+        let request = DotcomRequest(wordpressApiVersion: .wpcomMark2, method: .get, path: path, parameters: parameters)
+        let result: Result<[String: [String]], Error> = await enqueue(request)
+        let suggestions = (try? result.get())?["suggestions"] ?? []
+        return suggestions
+    }
+
+    public func createAccount(email: String,
+                              username: String,
+                              password: String,
+                              clientID: String,
+                              clientSecret: String) async -> Result<CreateAccountResult, CreateAccountError> {
+        let path = Path.accountCreation
+        let parameters: [String: Any] = [
+            "client_id": clientID,
+            "client_secret": clientSecret,
+            "signup_flow_name": "mobile-ios",
+            "flow": "signup",
+            "scheme": "woocommerce",
+            "password": password,
+            "email": email,
+            "username": username,
+            "validate": false,
```

Comment:
Nit: should we leave a short comment explaining why no validation is needed?

Reply:
good point, added a comment in https://github.com/woocommerce/woocommerce-ios/pull/7897/commits/0fb06e4c4ed12a2ceb478132f63240030ae1ea73